en/docs/chapter_sorting/merge_sort.md

```diff
@@ -2,19 +2,19 @@
 
 <u>Merge sort</u> is a sorting algorithm based on the divide-and-conquer strategy, involving the "divide" and "merge" phases shown in the figure below.
 
-1. **Divide phase**: Recursively split the array from the midpoint, transforming the sorting problem of a long array into that of shorter arrays.
-2. **Merge phase**: Stop dividing when the length of the sub-array is 1, start merging, and continuously combine two shorter ordered arrays into one longer ordered array until the process is complete.
+1. **Divide phase**: Recursively split the array from the midpoint, transforming the sorting problem of a long array into shorter arrays.
+2. **Merge phase**: Stop dividing when the length of the subarray is 1, and then begin merging. The two shorter sorted arrays are continuously merged into a longer sorted array until the process is complete.
 
 ![The divide and merge phases of merge sort](merge_sort.assets/merge_sort_overview.png)
 
 ## Algorithm workflow
 
-As shown in the figure below, the "divide phase" recursively splits the array from the midpoint into two sub-arrays from top to bottom.
+As shown in the figure below, the "divide phase" recursively splits the array from the midpoint into two subarrays from top to bottom.
 
-1. Calculate the midpoint `mid`, recursively divide the left sub-array (interval `[left, mid]`) and the right sub-array (interval `[mid + 1, right]`).
-2. Continue with step `1.` recursively until the sub-array interval length is 1 to stop.
+1. Calculate the midpoint `mid`, recursively divide the left subarray (interval `[left, mid]`) and the right subarray (interval `[mid + 1, right]`).
```

Comment:
The changes `subarray` `postorder` etc break the consistency.

<img width="362" alt="image" src="https://github.com/user-attachments/assets/1b27d5d5-f6c9-4932-9b67-2fa54812b3a6" />

Reply:
Please carefully review any modifications that may compromise the book’s consistency. If the changes are necessary, create a pull request to update all instances accordingly.

Reply:
Ok, I have changed them back